spec.bs

```diff
@@ -1948,9 +1970,57 @@ To <dfn>encode trusted signals keys</dfn> given an [=ordered set=] of [=strings=
 
 <div algorithm>
 
+To <dfn>convert an ad size to a string</dfn> given an [=ad size=] |adSize|.
+
+1. Let |sizeList| be an empty [=list=].
+1. [=list/Append=] |adSize|'s [=ad size/width=] to |sizeList|.

```

Comment:
Things like "width" are not strings at this point yet, right? `[=string/concatenating=]` seems to expect a list of all strings, so it's probably best to convert all non-string things to strings before appending them to `|sizeList|`.

Reply:
I have no idea what to reference to convert things to strings.  I added a "for each element / convert to string / replace with the result" step before concatenating, but I don't know what to reference for convert.  The infra spec all of the string/list operations are in has nothing like that.  Or should I explicitly make sizeList a sequence<USVString>, and through some magic say or imply each append statement should do the conversion,

Reply:
OK so what you're going to want to do here is:

1. Convert the Web IDL `double` to an ECMAScript value (see references like https://wicg.github.io/turtledove/#ref-for-dfn-convert-idl-to-javascript-value). This will invoke https://webidl.spec.whatwg.org/#ref-for-dfn-convert-idl-to-javascript-value%E2%91%A0%E2%91%A2, and give you a `Number` (ES type)
2. Call ECMAScript's `ToString()` on the newly-converted Number value. See https://github.com/whatwg/console/blob/main/index.bs#L18 for referencing this.

Reply:
Done, thanks!  I hadn't even realized that WebIDL doubles weren't a subset of ECMAScript Numbers.  Note that the bideshed server is down, so I can't currently verify this even compiles, let alone look at the results. :(

Reply:
It was almost there! I pushed up a small fix to your repo/branch and it all builds fine now (locally).